fedmsg.d/freshmaker-logging.py

```diff
@@ -3,15 +3,16 @@
         handlers={
             'console': {
                 'class': 'logging.StreamHandler',
-                'formatter': 'bare',
+                'formatter': 'json',
```

Comment:
@qixiang Should I also add the new json formatter instead of changing the bare one here?

Reply:
I think we don't need to add json formatter for development purpose, so the only change you need to do in this repo is  adding the `python3-json-logger` package to `yum-packages.txt`, so the image (used for our deployments) can be built with the new required rpm installed.